classes/submission/submissionfactory.php

```diff
@@ -41,18 +41,18 @@ public static function get_submission(int $coursemoduleid, int $userid) {
 
         // Throw exception if the course module is not found.
         if (empty($coursemodule)) {
-            throw new \moodle_exception('error:coursemodulenotfound', 'local_sitsgradepush', '', $coursemoduleid);
+            throw new \moodle_exception(
+                'error:coursemodulenotfound', 'local_sitsgradepush', '', null, "ID $coursemoduleid"
+            );
```

Comment:
Could remove the relative ```{$a}``` in the lang file.
https://github.com/ucl-isd/moodle-local_sitsgradepush/blob/CTP-3563-add-support-for-new-modules-marks-transfer/lang/en/local_sitsgradepush.php#L89

Reply:
Removed in commit cc1c7c0